docs/src/test/scala/docs/http/scaladsl/server/directives/JsonStreamingExamplesSpec.scala

```diff
@@ -85,10 +85,9 @@ class JsonStreamingExamplesSpec extends RoutingSpec {
     // {"example":43}
     // ...
     // {"example":1000}
-    val sep = ByteString("\n")
 
     implicit val jsonStreamingSupport = EntityStreamingSupport.json()
-      .withFramingRenderer(Flow[ByteString].mapConcat(_ :: sep :: Nil))
+      .withFramingRenderer(Flow[ByteString].map(bs => bs ++ ByteString("\n")))
```

Comment:
Create a single `ByteString` instance outside of the stream and reuse that instead of allocating a new one for every `map` call.